app/models/grda_warehouse/hud/client.rb

```diff
@@ -397,7 +397,7 @@ class Client < Base
     end
 
     scope :full_text_search, ->(text) do
-      text_search(text, client_scope: current_scope)
+      text_search(text, sorted: false)
```

Comment:
Oh, I missed that this dropped the `client_scope: current_scope` but text_search still uses it.  Let's add that back in.  (Unless there was a reason to remove it.)
```suggestion
      text_search(text, client_scope: current_scope, sorted: false)
```

Reply:
I took it out since current_scope is now the default for client_scope but it doesn't hurt to pass it in explicitly